src/pages/checkout.js

```diff
@@ -0,0 +1,93 @@
+import React, { useContext, useEffect, useState } from 'react';
+import { useNavigate } from 'react-router-dom';
+import { AppContext } from '../App';
+import { Title } from '../components/Products/productCardComponents';
+import {
+    CheckoutMainWrapper,
+    CheckoutDiv,
+    FormCheckout,
+    FormWrapper,
+    SummaryTableWrapper,
+    FormLabel,
+    FormInput,
+    FormTextarea,
+    StandardButton,
+    ButtonWrapper,
+    MainButtonWrapper,
+    CheckoutProduct,
+    CheckoutTotal,
+    Price
+} from './styled';
+
+const CheckoutPage = () => {
+    const navigate = useNavigate();
+    const { state } = useContext(AppContext);
+    const [cart, setCart] = useState([]);
```

Comment:
useReducer will help for larger state, or you can even set an object as state, this will make your component easier to read :)
`const [form, setForm] = useState({ name: '', email: ''})`...